cli/assets/generated/TestTemplate.t.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../src/{contract_name}.sol"; 
```

Comment:
For consistency with the book and [best practices](https://book.getfoundry.sh/tutorials/best-practices), we should use named imports here
```solidity
import {Test, console2} from "forge-std/Test.sol";
import {ContractName} from "../src/{contract_name}.sol"; 
```

Reply:
I agree. Can we just import `Test` from `forge-std/Test.sol`? I believe console2 is not being used in the generated file.

Reply:
> I believe `console2` is not being used in the generated file.

This is true, but it's convenient for users to have it imported by default. With `import "forge-std/Test.sol";` it's imported by default, but when you switch to named imports you have to include it explicitly

Reply:
That makes sense. Thanks for reviewing Matt! I have resolved it.